handler/create_scan_inference/create_scan_inference.go

```diff
@@ -10,8 +10,7 @@ import (
 )
 
 type createScanInferenceRequest struct {
-	ProjectId       *uint64                `json:"project_id"`
-	ImageUrl        string                 `json:"image_url"`
+	ScanId          *uint64                `json:"project_id"`
```

Comment:
Is a pointer necessary here? The size of the pointer is the same size of the integer itself

Reply:
I do a pointer for the sake of handling the nil case if the scanid is nil in the request. If it was just uint64, it would be 0 which could still be a valid id.

Reply:
Makes sense, LGTM